lib/runtime-core/src/fault.rs

```diff
@@ -25,13 +34,19 @@ use std::process;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::Once;
 
+#[cfg(target_arch = "x86_64")]
 pub(crate) unsafe fn run_on_alternative_stack(stack_end: *mut u64, stack_begin: *mut u64) -> u64 {
     raw::run_on_alternative_stack(stack_end, stack_begin)
 }
 
+#[cfg(not(target_arch = "x86_64"))]
+pub(crate) unsafe fn run_on_alternative_stack(stack_end: *mut u64, stack_begin: *mut u64) -> u64 {
+    unimplemented!("run_on_alternative_stack");
+}
+
 const TRAP_STACK_SIZE: usize = 1048576; // 1MB
 
-const SETJMP_BUFFER_LEN: usize = 27;
+const SETJMP_BUFFER_LEN: usize = 128;
 type SetJmpBuffer = [i32; SETJMP_BUFFER_LEN];
```

Comment:
Could I request an `#repr(align(`_x_`))` here? It might already be aligned enough, but I would really like it to be explicit so that it's clear that it's been checked, and in the future needs to be updated.